tests/test_sklearn_decision_tree_converters.py

```diff
@@ -142,6 +142,37 @@ def test_extra_trees_tree_trav_regressor_converter(self):
     def test_extra_trees_perf_tree_trav_regressor_converter(self):
         self._run_extra_trees_regressor_converter(1000, extra_config={"tree_implementation": "perf_tree_trav"})
 
+    # Used for DecisionTreeRegressor
+    def _run_decision_tree_regressor_converter(self, num_classes, extra_config={}):
+        # TODO: Extract method so that this can be reused across regressors.
+        warnings.filterwarnings("ignore")
+        for max_depth in [1, 3, 8, 10, 12, None]:
+            model = DecisionTreeRegressor(max_depth=max_depth)
+            X = np.random.rand(100, 200)
+            X = np.array(X, dtype=np.float32)
+            y = np.random.randint(num_classes, size=100)
+
+            model.fit(X, y)
+            pytorch_model = hummingbird.ml.convert(model, "pytorch", extra_config=extra_config)
+            self.assertTrue(pytorch_model is not None)
+            np.testing.assert_allclose(model.predict(X), pytorch_model.predict(X), rtol=1e-06, atol=1e-06)
+
+    # Regressor
+    def test_decision_tree_regressor_converter(self):
+        self._run_decision_tree_regressor_converter(1000)
```

Comment:
It may be worth adding a negative test case or two.

Reply:
What do you have mind as negative tests?

Reply:
I have only the test with unsupported extra config in mind. Unsure how useful it is.